lib/Notes/NotesAccordion/NotesAccordion.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component, Fragment } from 'react';
+import PropTypes from 'prop-types';
+import {
+  FormattedMessage,
+} from 'react-intl';
+import {
+  get,
+} from 'lodash';
+
+import {
+  Accordion,
+  Headline,
+  Badge,
+  Button,
+} from '@folio/stripes-components';
+
+import NotesList from './components';
+import styles from './NotesAccordion.css';
+
+export const noteShape = PropTypes.shape({
+  id: PropTypes.string,
+  lastSavedDate: PropTypes.instanceOf(Date),
```

Comment:
I think this shape should be moved to a separate file, since it's not specific to NotesAccordion and can be used in different components

Reply:
Will consider to move the shape out during adding of the other Notes specific components